src/components/Layout/Footer/Footer.tsx

```diff
@@ -1,167 +1,314 @@
-          target="_blank"
-          rel="noreferrer"
-        >
-          <Button
-            className={styles.antButton}
-            style={{ backgroundColor: '#CD201F' }}
-            icon={<YoutubeFilled className={styles.antIcon} />}
-          />
-        </a>
-        <a
-          href="https://join.slack.com/t/sefheadquarters/shared_invite/zt-1h5zt3go4-wnRDDpecbWiTdpDv1VUoVg"
-          target="_blank"
-          rel="noreferrer"
-        >
-          <Button
-            className={styles.antButton}
-            type="primary"
-            style={{ backgroundColor: '#3AAF85' }}
-            icon={<SlackOutlined className={styles.antIcon} />}
-          />
-        </a>
-        <a href="https://sef.discourse.group/" target="_blank" rel="noreferrer">
-          <Button
-            className={styles.antButton}
-            style={{ backgroundColor: '#5E72E4' }}
-            icon={
-              <img src="/discourse-icon.png" alt="discourse-icon" width={15} />
-            }
-          />
-        </a>
-      </div>
-    </Col>
+        </div>
+
+        {/* SEF Social Media Links */}
+        <div className="flex socialmedia mt-[10px]">
+          <div className="twitter w-7 h-7  rounded flex justify-center items-center bg-[#1DA1F2]">
+            <a href="https://twitter.com/goasksef">
+              <svg
+                className="w-6"
+                xmlns="http://www.w3.org/2000/svg"
+                aria-label="Twitter"
+                role="img"
+                viewBox="0 0 512 512"
+              >
+                <rect width="512" height="512" rx="15%" fill="#1da1f2" />
+                <path
+                  fill="#ffffff"
+                  d="M437 152a72 72 0 01-40 12a72 72 0 0032-40a72 72 0 01-45 17a72 72 0 00-122 65a200 200 0 01-145-74a72 72 0 0022 94a72 72 0 01-32-7a72 72 0 0056 69a72 72 0 01-32 1a72 72 0 0067 50a200 200 0 01-105 29a200 200 0 00309-179a200 200 0 0035-37"
+                />
+              </svg>
+            </a>
+          </div>
+
+          <div className="facebook w-7 h-7 rounded flex justify-center items-center  bg-[#3B5999] ml-2 mb-2">
+            <a href="https://www.facebook.com/sustainableeducationfoundation/">
+              <svg
+                className="w-7"
+                viewBox="0 -6 512 512"
+                xmlns="http://www.w3.org/2000/svg"
+              >
+                <path fill="#475a96" d="M0 0h512v500H0z" />
+                <path
+                  d="M375.717 112.553H138.283c-8.137 0-14.73 6.594-14.73 14.73v237.434c0 8.135 6.594 14.73 14.73 14.73h127.826V276.092h-34.781v-40.28h34.781v-29.705c0-34.473 21.055-53.244 51.807-53.244 14.73 0 27.391 1.097 31.08 1.587v36.026l-21.328.01c-16.725 0-19.963 7.947-19.963 19.609v25.717h39.887l-5.193 40.28h-34.693v103.355h68.012c8.135 0 14.73-6.596 14.73-14.73V127.283c-.001-8.137-6.596-14.73-14.731-14.73z"
+                  fill="#ffffff"
+                />
+              </svg>
```

Comment:
Let's create separate components for svgs so we can manage them easily. create a folder called assets->svgs and keep the svgs there.

Reply:
Sure, I will create